app/models/division.rb

```diff
@@ -114,4 +114,20 @@ def locale_names
       I18n.t("locale_name.#{locale}", locale: locale)
     end
   end
+
+  def quickbooks_connected?
+    self.class.connector.connected?
+  end
+
+  def quickbooks_connect(access_token:, params:)
+    self.class.connector.connect(access_token: access_token, params: params)
+  end
+
+  def quickbooks_disconnect
+    self.class.connector.disconnect
+  end
+
+  def self.connector
+    @connector ||= Accounting::Quickbooks::Connector.new Division.root.custom_data
```

Comment:
Also, `connector` is not a good method name because it doesn't reference quickbooks anywhere. Again, see above for a recommendation.